server/cmd/mmctl/commands/user.go

```diff
@@ -654,16 +656,19 @@ func deleteUsersCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 			continue
 		}
 		if res, err := c.PermanentDeleteUser(context.TODO(), user.Id); err != nil {
+			result = multierror.Append(result, err) 
 			printer.PrintError("Unable to delete user '" + user.Username + "' error: " + err.Error())
 		} else {
 			// res.StatusCode is checked for 202 to identify issues with file deletion.
 			if res.StatusCode == http.StatusAccepted {
+				err = errors.New("There were issues with deleting profile image of the user. Please delete it manually. Id: " + user.Id)
```

Comment:
Is this valid for only profile picture deletions? Might wanna be more general on the error?
```suggestion
				err = fmt.Errorf("deletion for user %s is partially done, check the server logs", user.Id)
```